frontend/src/Constants/ScreenSize.tsx

```diff
@@ -0,0 +1,7 @@
+const screenSizes = {
+  mobile: '320px',
+  tablet: '768px',
+  desktop: '1024px',
+} as const;
+export { screenSizes };
```

Comment:
It would give me this error if I did ***export default screensizes***
***No matching export in "src/Constants/ScreenSize.tsx" for import "screenSizes"***

Reply:
***export type ScreenSizes = typeof screenSizes;*** is needed for ***export { screenSizes }*** to work

Reply:
TypeScript thing. 😆 But if there's another way of doing this, then I bet Matthew will yell at me 👀

Reply:
I wouldn't really declare a type for this. It's just a constant object.